### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by vishal-biyani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 kbrew is to Kubernetes what Homebrew is to MacOS - a simple and easy to use package manager which hides the underlying complexity.
 
-Let's talk in context of an example of at installing Kafka on a Kubernetes cluster
- - You need cert manager & Zookeeper & kube-prometheus-stack for monitoring installed
- - Zookeeper is a operator so you need to create a CR of Zookeeper cluster after installation of operator.
+Let's talk in the context of an example of installing Kafka on a Kubernetes cluster
```
Comment (vishal-biyani) [inline README.md:9]:
```suggestion
Let's take the example of installing a Kafka on a Kubernetes cluster. If you are a developer trying this on a non prod environment you want a quick and simple way to set it up. But a typical process looks like this:
```

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ $ kbrew install kafka-operator
 ```
 ## Helm chart or operator or Manifests - all abstracted
 
-Kbrew abstracts the underlying chart or operator or manifest and gives you a recipe to install a stack with all basic configurations done.
+kbrew abstracts the underlying chart or operator or manifest and gives you a recipe to install a stack with all basic configurations done.
```
Comment (vishal-biyani) [inline README.md:23]:
```suggestion
kbrew abstracts the underlying chart or operator or manifest and gives you a recipe to install a stack with all specific configurations done. You can trust that the recipe `just works`!
```

File: README.md [hunk 3]
```diff
@@ -46,67 +46,122 @@ $ make
...
 Fetches updates for all the kbrew recipe registries
 
-### kbrew remove 
+#### kbrew remove 
 
-Partially implemented as of now.
+Uninstalls the application and its dependencies.
 
-## Terminology
+## Workflow
```
Comment (vishal-biyani) [inline README.md:102]: This should be called `Working of a Kbrew Recipe` or `Genesis of a Kbrew Recipe` 
Workflow is very overloaded and does not convey clearly what we want to say IMHO.

File: README.md [hunk 4]
```diff
@@ -46,67 +103,213 @@ $ make
 
 ## CLI Usage
 
-For a quick demo, watch: https://www.youtube.com/watch?v=pWRZhZgfYSw 
+```
+$ kbrew --help
+TODO: Long description
```
Comment (PrasadG193) [inline README.md:108]:
```suggestion
A Kubernetes tool that aims to make installing any complex stack in any cloud possible with one step.
```
This is getting updated with - https://github.com/kbrew-dev/kbrew/pull/109

File: README.md [hunk 5]
```diff
@@ -1,26 +1,83 @@
...
- - Zookeeper is a operator so you need to create a CR of Zookeeper cluster after installation of operator.
- - Then install Kafka operator
- - Create a CR of Kafka and wait for everything to stabilize.
- - Create ServieMonitor resources to enable prom scraping
 
-With kbrew all of this happens with a single command (This command will change in near future):
+![kbrew-logo](./images/kbrew-logo.png)
+
+
+kbrew is a Kubernetes tool that aims to make installing any complex stack in any cloud possible with `one step` (And yes we are definitely inspired by Homebrew from MacOS)
```
Comment (PrasadG193) [inline README.md:13]:
```suggestion
kbrew is a CLI tool for Kubernetes which makes installing any complex stack easy in `one step` (And yes we are definitely inspired by Homebrew from MacOS)
```

File: README.md [hunk 6]
```diff
@@ -1,26 +1,83 @@
+
```
Comment (PrasadG193) [inline README.md:1]:
```suggestion
```

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**